web/src/connectors/edit-model-connector/edit-model-connector.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { FC, useCallback, useState } from 'react';
+import EditModelSettings from 'components/model/add-model-settings/add-model-settings';
+import EditModelData from 'components/model/add-model-data/add-model-data';
+import { useEditModelMutation, useModels } from 'api/hooks/models';
+import { useCategories } from 'api/hooks/categories';
+import { Model, SortingDirection } from 'api/typings';
+import { useBasements } from 'api/hooks/basements';
+import Wizard, {
+    renderWizardButtons,
+    WizardPropsStep
+} from 'shared/components/wizard/wizard/wizard';
+import { MODELS_PAGE } from 'shared/constants/general';
+import { ModelValues } from '../../components/model/model.models';
+
+import { Form, IFormApi } from '@epam/uui';
+import wizardStyles from '../../shared/components/wizard/wizard/wizard.module.scss';
+
+type EditModelConnectorProps = {
+    onModelEdited: () => void;
+    onError: (err: any) => void;
+};
+
+const initialValues: ModelValues = {
+    name: undefined,
+    basement: undefined,
+    categories: undefined,
+    id: '',
+    score: undefined,
+    type: undefined,
+    training_id: undefined,
+    configuration_path_file: undefined,
+    configuration_path_bucket: undefined,
+    data_path_file: undefined,
+    data_path_bucket: undefined,
+    jobs: undefined
+};
+
+const EditModelConnector: FC<EditModelConnectorProps> = ({ onModelEdited, onError }) => {
+    const [stepIndex, setStepIndex] = useState(0);
+
+    const getMetadata = () => ({
+        props: {}
+    });
+
+    const { data: categories } = useCategories(
+        {
+            page: 1,
+            size: 100,
+            searchText: '',
+            sortConfig: { field: 'name', direction: SortingDirection.ASC }
+        },
+        {}
+    );
+
+    const { data: models } = useModels(
+        {
+            page: 1,
+            size: 100,
+            searchText: '',
+            sortConfig: { field: 'name', direction: SortingDirection.ASC },
+            filters: []
+        },
+        {}
+    );
+
+    const { data: basements } = useBasements(
+        {
+            page: 1,
+            size: 100,
+            searchText: '',
+            sortConfig: { field: 'name', direction: SortingDirection.ASC }
+        },
+        {}
+    );
+
+    const handleNext = () => {
+        setStepIndex(stepIndex + 1);
+    };
+    const handlePrev = () => {
+        setStepIndex(stepIndex - 1);
+    };
+    const finishButtonCaption = 'Add Model';
```

Comment:
I don't think that we need additional variable for this string

Reply:
I rewrote the component so that it can perform two functions - Add and Edit, and now this constant is needed